src/pages/membership.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react"
+import { PageProps, graphql } from "gatsby"
+import {
+    Button,
+    Hidden,
+    Theme,
+    createStyles,
+    withStyles,
+    WithStyles,
+} from "@material-ui/core"
+import { MembershipPageQuery } from "graphql-types"
+
+import SEO from "components/seo"
+import useBios from "hooks/useBios"
+import BioGrid from "components/Bios/BioGrid"
+import PageContent from "components/PageLayout/PageContent"
+import ParallaxBackground from "components/PageLayout/ParallaxBackground"
+import Section from "components/PageLayout/Section"
+import Text from "components/Typography/Text"
+import ArchiveSection from "components/Archive/ArchiveSection"
+
+const styles = (theme: Theme) =>
+    createStyles({
+        grid: {},
+        gridItem: {},
+    })
+
+type Props = WithStyles<typeof styles> &
+    PageProps & {
+        data: MembershipPageQuery
+    }
+
+function MembershipPage(props: Props) {
+    const { data, classes } = props
+    const { boardBackground } = data
+    if (!boardBackground) throw new Error("Board background does not exist.")
+
+    return (
+        <>
+            <SEO title="Membership Cards" />
+            <ParallaxBackground image={boardBackground}>
+                <Text variant="h3" color="white" align="center">
+                    TASA Membership Cards
+                </Text>
+                <Text align="center">Valid through the 2022-2023 school year</Text>
+            </ParallaxBackground>
+
+            <PageContent>
+                <Section title="What is a TASA Membership Card?">
+                    <Text align="center">
+                        A TASA membership card brings fun, happiness, 
+                        and premium benefits right to your door. These 
+                        benefits include 10% off Cuppa Tea, Class520, and Aroy 
+                        Thai (more to come!!). In addition, this card 
+                        will grant you free event tickets and raffle 
+                        tickets at our major events (Tour of Taiwan and 
+                        Night Market)! For all of these perks, you only 
+                        have to pay a small price of $15 or bundle with 
+                        some friends by buying a group package!
+                    </Text>
```

Comment:
Can we change the last 2 sentences to
"In addition, this card will award you free event tickets AND raffle tickets at two of our major events: Tour of Taiwan and Night Market! Receive all of these perks for the small cost of $15, or get some friends together and buy a group bundle!"